README.md

```diff
@@ -16,6 +16,12 @@ Run the (incomplete) test suite with
 $ make test
 ```
 
+Verify that the Sass can compile by running 
+
+```sh
+$ npx sass ./test/main.scss --load-path=./bower_components/
```

Comment:
Should this be `--include-path`?

Reply:
@i-like-robots at least npm's `sass`, which is now Dart Sass, uses `--load-path`/`-I`.
 `--include-path` is used in `node-sass`